### PR TITLE
feat(docs): add GID/UID changes to 1.22 release preview

### PR DESCRIPTION
**Pull request by maggieneterval**

### CODE REVIEW

File: community/releases/next-release-preview/index.md [hunk 1]
```diff
@@ -55,3 +55,12 @@ Spinnaker's UI has changed! An application's nested menus are now represented as
 This change should not introduce any interruptions to a vanilla install of `deck`. However, if you've already made navigational changes to your group's instance of `deck` or created custom banners/headers for your app, you may need to make updates. The pattern for creating new routes in the side nav can be observed in the feature's PR:
 
 https://github.com/spinnaker/deck/pull/8239
+
+### (Breaking Change) Spinnaker Dockerfile GID/UID changes
```
Comment (plumpy) [inline community/releases/next-release-preview/index.md]: I'd be inclined to put the breaking changes a the top, what do you think?

**Review by plumpy — APPROVED** [review]
Thanks! Gonna cross this off my todo list for today :grin:

File: community/releases/next-release-preview/index.md (hunk 1 above)
Comment (dorbin) [reply community/releases/next-release-preview/index.md]: +1

File: community/releases/next-release-preview/index.md [hunk 2]
```diff
@@ -55,3 +55,12 @@ Spinnaker's UI has changed! An application's nested menus are now represented as
...
+
+The Dockerfile of each Spinnaker microservice (except Halyard and Deck) now
+specifies an explicit GID and UID of `10111` for the `spinnaker` user.
+
+This is only a breaking change if you were relying on the previous
+non-deterministically assigned GID and UID
+(e.g., in a custom Kubernetes [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)).
```
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
(for example, in a custom Kubernetes [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)).
```

**Review by dorbin — APPROVED** [review]
👍

File: community/releases/next-release-preview/index.md (hunk 1 above)
Comment (maggieneterval) [reply community/releases/next-release-preview/index.md]: good call - done!

File: community/releases/next-release-preview/index.md (hunk 2 above)
Comment (maggieneterval) [reply community/releases/next-release-preview/index.md]: fixed, thanks!

**Pull request merged**